### DISCUSSION TITLE
Set LADP as default login option in zitadel

### DISCUSSION THREAD

**Discussion by SajjadManafi** [zitadel/zitadel#8847] [Q&A] [answered]
Hi everybody
Is there a solution to enable LADP as the default login option? (without disable username/password login)
In the current mode, login with username and password is default, and if the user wants to login with LADP, he can choose it on the login page.

**Comment by hifabienne** [comment, marked as answer]
At the moment this is not possible.
If you have one login possibility we show directly that possiblity.
As soon as you have multiple we show them on the username screen and the user has to chosse.
